src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/HostDocument.cs

```diff
@@ -21,10 +21,13 @@ public HostDocument(string filePath, string targetPath)
 
             FilePath = filePath;
             TargetPath = targetPath;
+            GeneratedCodeContainer = new GeneratedCodeContainer();
         }
 
         public string FilePath { get; }
 
         public string TargetPath { get; }
+
+        public GeneratedCodeContainer GeneratedCodeContainer { get; }
```

Comment:
whaaaa? When a project change the host documents make their way from one project config to the next, wouldn't that result in invalid generated code in docs? Maybe i'm missing something but this seems concerning.